var/spack/repos/builtin/packages/latte/package.py

```diff
@@ -17,6 +17,7 @@ class Latte(CMakePackage):
     tags = ['ecp', 'ecp-apps']
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
The reason for this is that once upon a time, we treated develop as newer than all releases, and master as older than all releases. This bug was fixed years ago, but there are still tons of packages that haven't been updated yet. The policy is to have them match to the best of our ability.

Reply:
Ok, got it.